tests/main/install-hook-can-run-svcs/test-snapd-install-hook-runs-svc/meta/hooks/install

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
+
+# start running
+snapctl start --enable "$SNAP_NAME.svc"
+
+# stop running
+snapctl stop --disable "$SNAP_NAME.svc"
```

Comment:
Silly question - isn't this racy wrt svc.sh?

Reply:
I second this question. Is `snapctl` synchronous?

Reply:
well snapctl is synchronous with systemd, but systemd is not synchronous with this snap actually creating that file, since the daemon is simple, as long as systemd can fork()/exec() the daemon successfully it considers it to have been started and as such would return to snapctl.

So as @stolowski points out this is racy.